Makefile

```diff
@@ -572,12 +573,18 @@ ifeq ($(L3_ENABLED), $(L3_DEFAULT))
 
 # Under L3-logging, invoke the L3-fast logging API, which needs the .S file
 ifeq ($(L3_FASTLOG_ENABLED), 1)
+
     CLIENT_SERVER_NON_MAIN_SRCS += $(L3_ASSEMBLY)
     CFLAGS += -DL3_FASTLOG_ENABLED
-endif
+
+else ifeq ($(L3_LOGT_FPRINTF), 1)
+
+    CFLAGS += -DL3_LOGT_FPRINTF
```

Comment:
Env-var `L3_LOGT_FPRINTF` translated down to compile code conditionally defined under `-DL3_LOGT_FPRINTF`.